index.html

```diff
@@ -14,34 +14,52 @@
     />
   </head>
   <body class="bg-img">
-    <header class="header">
+    <header class="header mobile-app">
       <nav>
         <ul  class="nav-list">
           <li><a class="nav-link" href="#">Hello</a></li>
           <li><a class="nav-link" href="#">Portfolio</a></li>
           <li><a class="nav-link" href="#">About</a></li>
           <li><a class="nav-link" href="#">Contact</a></li>
```

Comment:
- [ ] Please make sure that when the user clicks on any of the nav links, the page is scrolled to the part the user clicked on.